webrtc.html

```diff
@@ -6559,7 +6562,7 @@ <h2>Dictionary <code><a>RTCRtpHeaderExtensionCapability</a></code> Members</h2>
       <h3><dfn>RTCRtpReceiver</dfn> Interface</h3>
       <p>The <code>RTCRtpReceiver</code> interface allows an application to
       inspect the receipt of a <code>MediaStreamTrack</code>.</p>
-      <p>To <dfn>create an RTCRtpReceiver</dfn> with kind, <var>kind</var>, and
```

Comment:
My comment was in reference to this line, which uses `kind` as both type and name.